lib/compiler/ops-to-jsir.ts

```diff
@@ -306,6 +306,20 @@ export default class OpCompiler {
 
             this._irBuilder.popBlock(); // for-of
             this._irBuilder.popBlock(); // try-catch
+        } else if (expr instanceof Ops.ExistentialSubqueryBooleanExpressionOp) {
+            this._irBuilder.add(new JSIr.LoadConstant(new Ast.Value.Boolean(false), cond));
+
+            const blockStack = this._irBuilder.saveStackState();
+            const tmpScope = this._currentScope;
+
+            this._compileTable(expr.subquery);
+
+            this._irBuilder.add(new JSIr.LoadConstant(new Ast.Value.Boolean(true), cond));
+            this._irBuilder.add(new JSIr.Break());
+            this._irBuilder.popBlock();
```

Comment:
I'm not exactly sure this code works if the expression inside requires a reduction (eg aggregation, index, argmin/argmax). I think we need a test.

I'm also not exactly sure what the semantics of those would be inside an existential subquery. Maybe they need to be optimized out? Or maybe only index is preserved, and the semantics mean "at least this many rows were returned"?